src/main/resources/application.yml

```diff
@@ -2,5 +2,5 @@ app:
   datasource:
     jdbc-url: jdbc:postgresql://localhost:5432/gamevendor
     username: postgres
-    password: password
+    password: Jamkatel1@
```

Comment:
Not sure we need to change this as we will then need to change it all locally as well in docker.

Reply:
oh yeah, I previously set it up and the passwords differ. locally it is just different don't know how it got into the git